packages/components/src/Chip/Chip.css

```diff
@@ -1,9 +1,6 @@
-:root {
```

Comment:
We are attempting to clean up these `:root` declarations for component tokens, so I thought I'd do this while I'm in Chip.